mmdet/models/detectors/cascade_rcnn.py

```diff
@@ -57,19 +59,26 @@ def __init__(self,
                 self.bbox_head.append(builder.build_head(head))
 
         if mask_head is not None:
-            self.mask_roi_extractor = nn.ModuleList()
             self.mask_head = nn.ModuleList()
-            if not isinstance(mask_roi_extractor, list):
-                mask_roi_extractor = [
-                    mask_roi_extractor for _ in range(num_stages)
-                ]
             if not isinstance(mask_head, list):
                 mask_head = [mask_head for _ in range(num_stages)]
-            assert len(mask_roi_extractor) == len(mask_head) == self.num_stages
-            for roi_extractor, head in zip(mask_roi_extractor, mask_head):
-                self.mask_roi_extractor.append(
-                    builder.build_roi_extractor(roi_extractor))
+            assert len(mask_head) == self.num_stages
+            for head in mask_head:
                 self.mask_head.append(builder.build_head(head))
+            if mask_roi_extractor is not None:
+                self.shared_roi_extractor = False
```

Comment:
`shared_roi_extractor` is ambiguous, `share_roi_extractor` or `roi_extractor_shared` is better.